fv3core/stencils/xtp_u.py

```diff
@@ -16,82 +16,126 @@
 
 
 @gtscript.function
-def _get_flux(
+def advect_u(
```

Comment:
OK, did this refactor. Now bl_br are retrieved inside the nested function, and then since this function then becomes a one-liner calling another function, I deleted it and renamed advect_u to advect_u_along_x (commit b56f49cbb0a0ea4745c315438ca60014184eae65).